ldms/src/core/ldms.h

```diff
@@ -1206,6 +1218,30 @@ uint32_t ldms_set_perm_get(ldms_set_t s);
  */
 int ldms_set_perm_set(ldms_set_t s, mode_t perm);
 
+#define DEFAULT_AUTHZ_SET_UID 0x4
+#define DEFAULT_AUTHZ_SET_GID 0x2
+#define DEFAULT_AUTHZ_SET_PERM 0x1
+#define DEFAULT_AUTHZ_SET_ALL (SET_DEFAULT_AUTHZ_UID|SET_DEFAULT_AUTHZ_GID|SET_DEFAULT_AUTHZ_PERM)
+#define DEFAULT_AUTHZ_READONLY 0
+/**
+ * \brief Atomically set or get one or more default authorization values for LDMS sets.
+ *
+ * The uid, gid, and perm options can each individually be inputs or outputs. To make
+ * uid, gid, or perm as inputs set the bits DEFAULT_AUTHZ_SET_UID,
+ * DEFAULT_AUTHZ_SET_GID, or SET_DEFAULT_SET_AUTHZ_PERM bits respectively in the
+ * set_flags option.  DEFAULT_AUTHZ_SET_ALL is provided as a convience to set all three
```

Comment:
Just a typo here. "convience" should read "convenience."

Reply:
Yes, thanks.